DistributionPackages/Neos.NeosIo/Configuration/NodeTypes.VideoEmbed.yaml

```
@@ -0,0 +1,21 @@
+'Neos.NeosIo:VideoEmbed':
+  superTypes:
+    'Neos.Neos:Content': true
+  ui:
+    label: Video Embed
+    icon: icon-video
```

```suggestion
    icon: 'fas fa-play-circle'
```

I would here use a different icon, as this suggests more the autoplay feature. I don't know if `play-circle` is available, if not we could use the `play` icon